src/CLICommand/parameters_helper.ts

```diff
@@ -10,14 +10,16 @@ import {
 	MaxDepthParameter,
 	SeedPhraseParameter,
 	WalletFileParameter,
-	PrivateParameter
+	PrivateParameter,
+	BoostParameter
 } from '../parameter_declarations';
 import { cliWalletDao } from '..';
 import { DriveID, DriveKey } from '../types';
 import passwordPrompt from 'prompts';
 import { PrivateKeyData } from '../private_key_data';
 import { ArweaveAddress } from '../types/arweave_address';
 import { SeedPhrase } from '../types/seed_phrase';
+import { FeeMultiple } from '../types/';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type ParameterOptions = any;
```

Comment:
To-Do for @matibat: a dict is more specific than `any`

```suggestion
type ParameterOptions = Register<string, unknown>;
```

Reply:
And when there is a `ParameterName` type, I'd suggest:
```suggestion
type ParameterOptions = Register<ParameterName, unknown>;
```